wandb: Using wandb-core as the SDK backend.  Please refer to https://wandb.me/wandb-core for more information.
wandb: Currently logged in as: adiasija10. Use `wandb login --relogin` to force relogin
wandb: Tracking run with wandb version 0.19.0
wandb: Run data is saved locally in /home/aasija1/temp_run_wavelets/wandb/run-20241204_220709-x5xnha63
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run pretty-bee-6
wandb: ‚≠êÔ∏è View project at https://wandb.ai/adiasija10/wavelet-cnn
wandb: üöÄ View run at https://wandb.ai/adiasija10/wavelet-cnn/runs/x5xnha63
/home/aasija1/.conda/envs/env2/lib/python3.10/site-packages/torch/utils/data/dataloader.py:617: UserWarning: This DataLoader will create 4 worker processes in total. Our suggested max number of worker in current system is 1, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
  warnings.warn(
Using downloaded and verified file: /home/aasija1/.medmnist/pathmnist.npz
Using downloaded and verified file: /home/aasija1/.medmnist/pathmnist.npz
len train loader:2813 , validation loader :313
Epoch 1/20
----------
Traceback (most recent call last):
  File "/home/aasija1/temp_run_wavelets/main.py", line 29, in <module>
    model = train_model(model, dataloaders, criterion, optimizer, num_epochs, device)
  File "/home/aasija1/temp_run_wavelets/training.py", line 29, in train_model
    loss = criterion(outputs, labels)
  File "/home/aasija1/.conda/envs/env2/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1736, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/aasija1/.conda/envs/env2/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1747, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/aasija1/.conda/envs/env2/lib/python3.10/site-packages/torch/nn/modules/loss.py", line 1293, in forward
    return F.cross_entropy(
  File "/home/aasija1/.conda/envs/env2/lib/python3.10/site-packages/torch/nn/functional.py", line 3479, in cross_entropy
    return torch._C._nn.cross_entropy_loss(
RuntimeError: 0D or 1D target tensor expected, multi-target not supported
[1;34mwandb[0m: üöÄ View run [33mpretty-bee-6[0m at: [34mhttps://wandb.ai/adiasija10/wavelet-cnn/runs/x5xnha63[0m
[1;34mwandb[0m: Find logs at: [1;35mwandb/run-20241204_220709-x5xnha63/logs[0m
